egs/chime6/s5_track2/local/diarize.sh

```diff
@@ -85,29 +86,35 @@ if [ $stage -le 4 ]; then
   echo "$0: wrote RTTM to output directory ${out_dir}"
 fi
 
+hyp_rttm=${out_dir}/rttm
```

Comment:
This should be {data_in}/rttm?

Reply:
I don't see why? This is the location of the newly generated hypothesis RTTM file.

Reply:
You are right. Sorry I made a wrong thought.